app/views/course/sections/support-and-advice.njk

```diff
@@ -1,11 +1,10 @@
 <div class="govuk-!-margin-bottom-8">
   <h2 id="support-and-advice" class="govuk-heading-l">Support and advice</h2>
-  <p class="govuk-body">For support and advice, you can speak to a <a class="govuk-link" href="#">teacher training adviser</a> for free. They’re all experienced teachers who can help you to prepare your application, book school
-    experience, and access exclusive teaching events.</p>
+  <p class="govuk-body">You can speak to a <a class="govuk-link" href="#">teacher training adviser</a> for free, dedicated support. They’re all experienced teachers who can give you one-to-one support with your application, help you get school
+    experience, and find events where you can speak to training providers and teachers.</p>
```

Comment:
> They‘re all **experienced** teachers who can give you **experience**, and find events where you can speak to training providers and teachers.

You've repeated the word 'experience', with the second instance not making much sense. Could the second instance be replaced with 'advice'?

> They‘re all experienced teachers who can give you **advice**, and **help you** find events where you can speak to training providers and teachers.

(emphasis mine)